Makefile

```diff
@@ -212,16 +212,20 @@ ui-pkg-upload: tmp/weave-scope.tgz
 
 clean:
 	$(GO) clean ./...
-# Don't actually rmi the build images - rm'ing the .uptodate files is enough to ensure
```

Comment:
keep this comment but add 'in most cases'. below, add a comment on why you sometimes do need realclean.